composer.json

```diff
@@ -77,7 +78,7 @@
             "./bin/console lint:twig ./templates",
             "./bin/console lint:yaml ./config",
             "minus-x check .",
-            "phan --allow-polyfill-parser",
+            "phan --allow-polyfill-parser --long-progress-bar",
```

Comment:
Why add this? (Not that it matters, seems sensible; just wondering!)

Reply:
I was originally under the impression that the default would be to use no progress bar at all. Then I realized that `long-progress-bar` is actually the default, but I figured I would leave it anyway so it's more explicit.

As for why this specific choice: I think a progress bar is always necessary for this kind of tools, since they might become slow-ish, and the other progress bar (that you'd get with `-p`) is not really suitable for CI logs because it takes up a lot of space.